### PR TITLE
docs: clarified restricted environment scenario

### LINKED ISSUE #2794: KinD installer/Resource not present
### Bug description

KinD installer/Resource not present therefore I'm not able to create a kubernetes cluster.

The "KinD Extension" is installed and status runningrnetes cluster with Kind in podman-desktop.

Podman-desktop: 1.0.1 and 1.1.1.0
Podman: 4.5.0 and 4.5.1

image

image

image

### Operating system

Windows 10.0.19045 (22H2) (amd64)

### Version

1.1.0

### Steps to reproduce

_No response_

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

docs: clarified restricted environment scenario

### What issues does this PR fix or reference?

references #2794

### CODE REVIEW

File: website/docs/Installation/windows-install/installing-podman-desktop-and-podman-in-a-restricted-environment.md [hunk 1]
```diff
@@ -33,10 +34,12 @@ The Windows installer for restricted environments:
 #### Procedure
 
 1. [Download the Windows installer for restricted environments](/downloads), and copy the downloaded file to the Windows host in a restricted environment.
-2. Run the Windows installer for restricted environments to install Podman Desktop.
-3. Open Podman Desktop and click on the **Install** button to install Podman.
-4. Click on **Initialize Podman**.
+1. Run the Windows installer for restricted environments to install Podman Desktop.
+1. Open Podman Desktop and click on the **Install** button to install Podman.
```
Comment (themr0c) [inline website/docs/Installation/windows-install/installing-podman-desktop-and-podman-in-a-restricted-environment.md]:
```suggestion
1. Open Podman Desktop and click the **Install** button to install Podman.
```

**Review by cdrage — DISMISSED** [review]
1 weird nitpick, `Installation` folder is capitalized. Should be just `website/docs/installation`

Otherwise, the docs lgtm!

**Comment by cdrage** [comment]
There is also vale errors for a broken link

**Comment by cdrage** [comment]
Uh oh. Mermaid is not working.

The screenshot is here:
Screenshot 2023-09-06 at 11 15 41 AM


According to the issue: https://github.com/facebook/docusaurus/issues/9032 this is being worked on and planned for the next release (3.0).

So for the time being, can you please take a screenshot of the diagram + add this to the PR?

We should open up an issue for the time being to re-enable mermaid js when docusaurus has a fix.

**Comment by themr0c** [comment]
- That's strange, the mermaid graph worked on my end.

- The `Installation` directory is like that already. It will need  a separate pull request for that. I am not sure yet if it requires rewrites or not.
- Vale error fixed.
- Build error: requires the compose pull request to get merged first.

**Comment by cdrage** [comment]
LGTM! 

We'll change `Installation` to `installation` in a separate PR, you're right.

And let's get https://github.com/containers/podman-desktop/pull/3716 in so Vale issues are passed.

**Comment by themr0c** [comment]
Fixed link, please approve and merge.

**Review by cdrage — APPROVED** [review]
LGTM!

**Pull request merged**